packages/suite-native-navigation/src/screens/DummyScreen.tsx

```diff
@@ -12,7 +12,7 @@ const backgroundStyle = prepareNativeStyle<{ isDarkMode: boolean }>(
     }),
 );
 
-export const Home = () => {
+export const DummyScreen = () => {
```

Comment:
Please rename this to `DemoScreen` and rename this whole folder to `dummyScreens` to make it obvious that no one should add anything serious here

Reply:
demo, dummy, hodinky, holinky... :D Ok, renamed in https://github.com/trezor/trezor-suite/commit/586add7a095903bc6b92f49f63f61c3c7927d312